.github/CODEOWNERS

```diff
@@ -13,8 +13,22 @@
 # the core team as a whole will be assigned
 *       @dbt-labs/core
 
-# Changes to GitHub configurations including Actions
-/.github/ @leahwicz
+### OSS Tooling Guild
+
+/.github/                       @dbt-labs/guild-oss-tooling
+.bumpversion.cfg                @dbt-labs/guild-oss-tooling
+
+/.changes/                      @dbt-labs/guild-oss-tooling
+.changie.yaml                   @dbt-labs/guild-oss-tooling
+
+pre-commit-config.yaml          @dbt-labs/guild-oss-tooling
+pytest.ini                      @dbt-labs/guild-oss-tooling
+tox.ini                         @dbt-labs/guild-oss-tooling
+
+pyproject.toml                  @dbt-labs/guild-oss-tooling
+requirements.txt                @dbt-labs/guild-oss-tooling
+dev_requirements.txt            @dbt-labs/guild-oss-tooling
+/core/setup.py                  @dbt-labs/guild-oss-tooling
```

Comment:
Should `/plugins/postgres/setup.py` and `/core/MANIFEST.in` be in this list too?
I see in the root level [requirements.txt](https://github.com/dbt-labs/dbt-core/blob/main/requirements.txt) a reference to:
- `./core` that points pip to: 
   - `/core/setup.py` (covered here) 
   - `/core/MANIFEST.in` (does this belong here?)
- `./plugins/postgres` that points pip to `/plugins/postgres/setup.py`

Reply:
Good catch!  Added.  

`/plugins/postgres/setup.py` is listed below with two owners.  The adapters team wants to maintain ownership of the file as well.